base/deprecated.jl

```diff
@@ -80,7 +80,7 @@ function firstcaller(bt::Array{Ptr{Void},1}, funcsym::Symbol)
         if lkup === ()
             continue
         end
-        fname, file, line, fromC = lkup
+        fname, file, line, inlined_file, inlined_line, fromC = lkup
```

Comment:
Yes, there is no separate frame ip, but there is a separate source location record (conceptually speaking).

I still don't quite understand this. The printing code shows `inlinedfile : inlinedline`, *then* says `by code inlined from: ` and shows `file : line`. That seems to contradict inlinedfile/inlinedline meaning where the code was inlined from. Or I might be misinterpreting something.